sigstore/_cli.py

```diff
@@ -692,3 +719,35 @@ def _get_identity_token(args: argparse.Namespace) -> Optional[str]:
             issuer,
         )
     return token
+
+
+class SplitCertificateChainError(Exception):
+    pass
+
+
+def _split_certificate_chain(chain_pem: str) -> List[bytes]:
```

Comment:
What do you think about moving this helper + exception to the `_utils` module? We already have a couple of PEM helpers in there, and it would make unit testing a little easier.

Reply:
[da25f55](https://github.com/sigstore/sigstore-python/pull/323/commits/da25f55dbf89d8b906205a9dad4707c92f21939a)